Directory.Packages.props

```
@@ -24,7 +24,7 @@
     <PackageVersion Include="xunit.assert" Version="2.4.1" />
     <PackageVersion Include="xunit.extensibility.core" Version="2.4.1" />
     <PackageVersion Include="xunit.extensibility.execution" Version="2.4.1" />
-    <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.3" />
+    <PackageVersion Include="xunit.runner.visualstudio" Version="3.1.1" />
```

This version has some bugs that were addressed in 3.1.3

```suggestion
    <PackageVersion Include="xunit.runner.visualstudio" Version="3.1.1" />
```